### PR TITLE
Feature capstone

### PR DESCRIPTION
**Pull request by gak92**
- Build complete metrics web-app project using React/Redux
- Follow Git-flow
- Used React Components and Routes
- Used Redux store
- Used React-bootstrap for styling
- Fetch Data from API (https://site.financialmodelingprep.com/developer/docs)
- Used async-thunk for async operation calls

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +72,4 @@ Give a ⭐️ if you like this project!
```
Comment (Wes-Isaac) [inline README.md:73]: - [ ] Great job so far, one of the requirement is that it contains an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio), kindly add that.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,58 @@
...
+- GitFlow
+- Webpack
+- Jest Testing
+- Netlify
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
 
+[Live Demo](https://stock-metrics-webapp.netlify.app/)
+[Video Demo](https://www.loom.com/share/d816f2ac60b042b0b0c360c7a366e2a3)
```
Comment (Wes-Isaac) [inline README.md]: - [ ] Nice presentation, in the video please add a piece of code that you're most proud of as stated in the requirements.

Screen Shot 2022-09-02 at 1 02 37 PM

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @gak92, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 2 above)
Comment (gak92) [reply README.md]: @Wes-Isaac,
Thanks for the review
- Updated Readme to add Acknowledgement of the designer
- Also recreate the video with highlight the section on which I am proud of.

**Review by julie-ify — APPROVED** [review]
Hi @gak92,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**